### DISCUSSION TITLE
Chrome custom window titles

### DISCUSSION THREAD

**Discussion by jennokie37** [lawand-dot-io/taskbar#116] [Feature requests]
**Describe the bug**
My computer did an update, and the change appeared. My tabs for Chrome at the bottom were all named using Window--Name Window, and now the names are gone. I have tried restarted Taskbar, Chrome and the computer itself to no avail.

**Expected behavior**
The tabs should read the label that I created for them so that I know what each one is. 

**Desktop (please complete the following information):**
 - Taskbar version: 1.2.5
 - OS: macOS 15.1.1
 - Chip: Apple M3

**Comment by lawand-dot-io** [comment]
Hello. I haven't tried this Chrome feature before. Which version of macOS did you have before the update? macOS 14?

**Comment by lawand-dot-io** [comment]
As far as I can see, chrome is responsible for the custom window titles as this is a chrome feature and not a taskbar feature. I will convert this ticket from a bug report to a feature request so that I try to investigate it more in the future. Cheers

**Comment by lawand-dot-io** [comment]
Related: https://github.com/lawand-dot-io/taskbar/discussions/123

**Comment by lawand-dot-io** [comment]
Related: https://github.com/lawand-dot-io/taskbar/issues/151
